enrolments/tests/views/test_classes.py

```diff
@@ -0,0 +1,108 @@
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from accounts.models import User
+from enrolments.models import Session, Class
+from enrolments.serializers import (
+    SessionSerializer,
+    ClassListSerializer,
+    SessionDetailsSerializer,
+)
+
+
+class SessionTestCase(APITestCase):
+    def setUp(self):
+        self.user = User.objects.create(email="user@staff.com")
+        self.session1 = Session.objects.create(season=Session.SPRING, year=2021)
+        self.session2 = Session.objects.create(season=Session.FALL, year=2020)
+        self.class1 = Class.objects.create(
+            name="class1", session=self.session1, facilitator=self.user
+        )
+        self.class2 = Class.objects.create(
+            name="class2", session=self.session1, facilitator=self.user
+        )
+        self.class3 = Class.objects.create(
+            name="class3", session=self.session2, facilitator=self.user
+        )
+
+    def test_get_all_sessions(self):
```

Comment:
I think this test already exists in `test_session.py`.. unless I understood this wrong

Reply:
Good catch, removing tests that have already been included in `test_session`.